include/boost/compute/algorithm/random_shuffle.hpp

```diff
@@ -46,7 +50,13 @@ inline void random_shuffle(Iterator first,
     // generate shuffled indices on the host
     std::vector<cl_uint> random_indices(count);
     boost::iota(random_indices, 0);
+#ifdef BOOST_COMPUTE_USE_CPP11
+    std::random_device nondeterministic_randomness;
+    std::default_random_engine random_engine(nondeterministic_randomness);
```

Comment:
Shouldn't it be `std::default_random_engine random_engine(nondeterministic_randomness());`?